includes/utilities.inc

```diff
@@ -908,21 +908,22 @@ function islandora_paged_content_pdfinfo_availability() {
  *   The page number to extract it will be used as the first and last page
  *   value. Note that pdftotext won't complain about using an offset greater
  *   than the number of allotted pages of a PDF.
- *
+ * @param bool $raw_text
+ *   Whether to extract the raw text from the PDF or not.
  * @param bool $layout
  *   Whether we are retaining layout of the PDF or not.
  *
  * @return bool|string
  *   FALSE if it fails, the URI of the text file otherwise.
  */
-function islandora_paged_content_extract_text_from_pdf($uri, $offset, $layout = TRUE) {
+function islandora_paged_content_extract_text_from_pdf($uri, $offset, $raw_text = FALSE, $layout = TRUE) {
```

Comment:
@jordandukart

The new var should be pushed to the end to preserve backwards compatibility.

Thanks!

----
William Panting
Developer
discoverygarden inc. | Managing Digital Content